<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sensor</class>
 <widget class="QWidget" name="Sensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gpsDash">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="horizontalSpacing">
     <number>14</number>
    </property>
    <property name="verticalSpacing">
     <number>45</number>
    </property>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="layout_Settings" stretch="0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QGridLayout" name="layout_Serial">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="serial_connect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>361</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Serial Connections</string>
          </property>
          <widget class="QComboBox" name="portSelect">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>30</y>
             <width>79</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="serialConnectionButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>80</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
          <widget class="QPushButton" name="serialTerminationButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>90</y>
             <width>80</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
          <widget class="QLabel" name="port">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>41</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Port </string>
           </property>
          </widget>
          <widget class="QLabel" name="baud">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>30</y>
             <width>81</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Baud rate</string>
           </property>
          </widget>
          <widget class="QComboBox" name="baudSelect">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>30</y>
             <width>86</width>
             <height>25</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>110</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>300</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>128000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>256000</string>
            </property>
           </item>
          </widget>
          <widget class="QRadioButton" name="saveButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>141</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Save Data</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="gps_data_display">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>GPS Data</string>
          </property>
          <widget class="QTextBrowser" name="textbrowser">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>343</width>
             <height>192</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="gpstype">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>341</width>
             <height>21</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="local_data">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>110</height>
           </size>
          </property>
          <property name="title">
           <string>Read Local Data</string>
          </property>
          <widget class="QLabel" name="path">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>41</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Path</string>
           </property>
          </widget>
          <widget class="QTextBrowser" name="pathBrowser">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>30</y>
             <width>291</width>
             <height>41</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="serialLocalConnectionButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>80</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
          <widget class="QPushButton" name="serialLocalTerminationButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>80</y>
             <width>80</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="layout_Display" stretch="0">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedDisplay">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="Map">
         <property name="contextMenuPolicy">
          <enum>Qt::PreventContextMenu</enum>
         </property>
         <widget class="QWidget" name="layoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>899</width>
            <height>651</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="layout_map">
           <item>
            <widget class="QLabel" name="Label_LiveRocketView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>1209</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Live Rocket Position</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bullshit_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>  Map View Widget Goes Here                                                    </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="next">
          <property name="geometry">
           <rect>
            <x>810</x>
            <y>10</y>
            <width>61</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="Graph">
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>881</width>
            <height>651</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="layout_graph">
           <item>
            <widget class="QLabel" name="Label_LiveRocketData">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Live Rocket Data</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="graph_display">
             <property name="spacing">
              <number>6</number>
             </property>
             <item row="0" column="0" rowspan="2">
              <layout class="QVBoxLayout" name="layout_altitude">
               <item>
                <widget class="QLabel" name="altitude">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Altitude (1 Unit = 1m)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCustomPlot" name="graph_alt" native="true"/>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="layout_x_axis">
               <item>
                <widget class="QLabel" name="x_axis">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Latitude</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCustomPlot" name="graph_x" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>875</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <layout class="QVBoxLayout" name="layout_y_axis">
               <item>
                <widget class="QLabel" name="y_axis">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Longitude</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCustomPlot" name="graph_y" native="true"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="prev">
          <property name="geometry">
           <rect>
            <x>810</x>
            <y>10</y>
            <width>61</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>Prev</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">src/graph/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
